### ISSUE TITLE
Change magic, enigmatic constants into text defines.

### ISSUE THREAD

**Issue by dmarszk** [Wisling/tibiaauto#10]
Many constants through code (most notably, DLL pipe protocol) aren't defined, which makes code much harder to understand, and is more error-prone.

**Comment by Wisling** [comment]
Lol indeed!  Done a bit of this, but always on code that I was going to change anyhow.  If you see an area that's a mess feel free to reformat it, though always ensure it does the same thing as it did before :P

**Comment by Wisling** [comment]
Question about this if you still remember what it is.  Do you mean that when we send a message through a pipe to tibiaautoinject2, for example <charname 32 bytes>, <text length 4 bytes>, <text variable bytes>, it gets defined separately in two places, the sending dll and again in tibiaautoinject2?

**Comment by dmarszk** [comment]
Yeah the pipe protocol is one of the worst things to understand, debug and update. That's the major thing that hurts my eyes. However, it works as is. So I'd consider the issue being a low priority.
